.test-infra/jenkins/job_LoadTests_coGBK_Python.groovy

```diff
@@ -25,7 +25,7 @@ import InfluxDBCredentialsHelper
 
 def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
 
-def loadTestConfigurations = { datasetName ->
+def loadTestConfigurations = { mode, datasetName ->
```

Comment:
I think we should agree on the common name of this param. In combine I put `jobType` (mainly because Java test has `jobType`), but there is a possibility to change it since combine isn't merged yet.

Reply:
After an offline discussion, we decided to stay with `mode` in this context.